pkg/controller/deployment/deployment_controller.go

```diff
@@ -796,5 +797,6 @@ func (dc *DeploymentController) checkForProgress(key string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	glog.V(2).Infof("Syncing deployment %q for a progress check", key)
```

Comment:
As often as "Queueing up deployment..." in progress.go. That shouldn't be that often - depends on the # of pods and it's v2. We can raise the lvl later, now I want them purely for debugging an ongoing flake that is blocking the merge queue (https://github.com/kubernetes/kubernetes/issues/39785).